src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -865,6 +960,12 @@
      */
     public static final Key<Value<Boolean>> SUSPENDED = KeyFactory.fake("SUSPENDED");
 
+    /**
+     * Represents the {@link Key} for representing the scoreboard tags applied
+     * to an entity.
```

Comment:
I would link to the wiki, but currently the only thing on them there is the command syntax, no actual description or anything, so it doesn't seem that useful.